docs/guides/device-gateway/linux.md

```diff
@@ -86,7 +86,9 @@ ngrok tcp 22 --cidr-allow ALLOWED_IP_ADDRESS_CIDR
 Alternatively, you can create an IP policy in the ngrok dashboard (under [Security > IP Restrictions](https://dashboard.ngrok.com/security/ip-restrictions)), and leverage the same policy to control access to your entire device fleet.
 :::
 
-## Step 4: Configure ngrok to recover on outages
+## Step 4: Run ngrok as a service so you don't need to manually restart
+
+Read more about the `ngrok service`command which allows you to run and control an ngrok service on the operating system. [Run ngrok as a service.](<[https://dashboard.ngrok.com/cloud-edge/tcp-addresses](https://ngrok.com/docs/agent/cli/#ngrok-service)>).
```

Comment:
```suggestion
Read more about the `ngrok service`command which allows you to run and control an ngrok service on the operating system. [Run ngrok as a service.](/agent/cli/#ngrok-service)).
```
@samcrichard is this more what you meant to do here? Or did you want to point to the TCP addresses page